cmd/minishift/cmd/start_preflight.go

```diff
@@ -263,9 +270,33 @@ func checkLibvirtInstalled() bool {
 	return true
 }
 
-//checkLibvirtDefaultNetwork returns true if the "default" network is present and active
-func checkLibvirtDefaultNetwork() bool {
+//checkLibvirtDefaultNetworkExists returns true if the "default" network is present
+func checkLibvirtDefaultNetworkExists() bool {
+	cmd := exec.Command("virsh", "--connect", "qemu:///system", "net-list")
```

Comment:
@gbraad Shouldn't it be `LC_ALL=en_us.UTF8 virsh --connect qemu:///system net-list`?

Reply:
That will result in `LC_ALL=en_US.UTF8 is not a command`. This is not a shell call, but just a process invocation.